src/pages/Clusters.js

```diff
@@ -157,6 +157,7 @@ const Clusters = () => {
   ])
 }
 
+// FIXME: do I want to change these?
```

Comment:
expand here to see several cases where I think there needs to be some conversation if we want to change navigation paths

Reply:
See above, let's leave the path intact.